app/views/sign_up_sheet/_table_header.html.erb

```diff
@@ -10,6 +10,7 @@
   <th width="10%">Bookmarks</th>
   <th width="10%">Actions</th>
   <th width="5%">Advertise- ment(s)</th>
+  <th width="5%">Link</th>
```

Comment:
could you make the link icon show next to "more..." as part of  "Topic name(s)" column. In this way a column will be saved. Thanks.